src/libraries/System.Runtime.InteropServices.JavaScript/tests/System.Runtime.InteropServices.JavaScript.UnitTests/System/Runtime/InteropServices/JavaScript/WebWorkerTestBase.cs

```diff
@@ -156,14 +156,20 @@ public class NamedCall
                     using var cts = new CancellationTokenSource(8);
                     try {
                         mr.Wait(cts.Token);
-                    } catch (OperationCanceledException) { /* ignore */ }
+                    } catch (OperationCanceledException) { 
```

Comment:
it should not be swallowed I think. Because it's not really the expected outcome.
If it was expected outcome, the outer assert should process that case, rather than misleading null, which for this test means that something is broken for Main and WebWorker executors.

Reply:
cc @radekdoulik

Reply:
Updated to not swallow `OperationCanceledException`.